rapido.css

```diff
@@ -5,7 +5,7 @@
 }
 
 body.rapido {
-	font-family: "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+	font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
```

Comment:
Are you sure about this? We will have different renderings with different fonts on different systems, and therefore we will not have a consistent user experience

Reply:
Nevermind, I have seen the links in the issue